app/src/main/res/layout-land/login_tout_layout.xml

```diff
@@ -1,74 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView
```

Comment:
What has changed on this file? I've reviewed and seem just identation changes, have I missed something @hadia